oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/query/ElasticRequestHandler.java

```diff
@@ -189,6 +204,77 @@ private boolean requiresFacets() {
                 .map(pd -> pd.name);
     }
 
+    /*
+    Generates mlt query builder from the given mltQueryString
+    There could be 2 cases here -
+    1) select [jcr:path] from [nt:base] where similar(., '/test/a') [Return nodes with similar content to /test/a]
+    Xpath variant - //element(*, nt:base)[rep:similar(., '/test/a')]
+    In this case org.apache.jackrabbit.oak.query.ast.SimilarImpl creates the mltQueryString as
+    mlt?mlt.fl=:path&mlt.mindf=0&stream.body=/test/a
+    2) select [jcr:path] from [nt:base] where " +
+       "native('elastic-sim', 'mlt?stream.body=/test/a&mlt.fl=:path&mlt.mindf=0&mlt.mintf=0')
+       In this case the the exact mlt query passed above is passed to this method. This can be useful if we want to
+       fine tune the various default parameters.
+       The function name passed to native func ('elastic-sim') needs to be defined on index def
+       Refer https://jackrabbit.apache.org/oak/docs/query/lucene.html#native-query
+       TODO : Docs for writing a native mlt query with the various parameters that can be tuned
+       (The above is important since this is not a one-size-fits-all situation and the default values might not
+       be useful in every situation based on the type of content)
+     */
+    private QueryBuilder moreLikeThisQuery(String mltQueryString) {
+        MoreLikeThisQueryBuilder mlt = null;
```

Comment:
minor: `null` is redundant
```suggestion
        MoreLikeThisQueryBuilder mlt;
```